test/functional/insert.test.js

```diff
@@ -3,15 +3,15 @@ const test = require('./shared').assert,
   setupDatabase = require('./shared').setupDatabase,
   Script = require('vm'),
   expect = require('chai').expect,
-  normalizedFunctionString = require('bson/lib/bson/parser/utils').normalizedFunctionString,
+  normalizedFunctionString = require('bson/lib/parser/utils').normalizedFunctionString,
```

Comment:
feel free when editing sections like this to introduce modern features like object destructuring.